README.md

```diff
@@ -21,3 +21,23 @@ Then, use the included bash script to automatically re-run generation of the doc
 ./runDevelop.sh
 ```
 
+## Source repositories
+
+The documentation is building by using documentation sources from multiple repositories.
+You can look at the [playbook](./antora-playbook.yml) to see the up-to-date list of repositories.
+
+##  Versions
+
+The documentation is built for multiple versions.
+The latest version named `main` is built from the `HEAD` of the `main` branches of all source repositories.
+An additional version is built for branch matching `docs/release/<version>`.
+
+For example, from the branches (in this and all source repositories) named `docs/release/23.1` a version named `23.1` will be created.
+
+
+### Adding a new version
+
+If you want to add a new version,
+create an appropriately named branch in this repository **and all source repositories**
+pointing to the commit you want the documentation
+to be fetched from.
```

Comment:
Maybe one line to match the previous style of [one sentence per line](https://sive.rs/1s)?

Reply:
Done.

Note: This long sentence was previously broken up using [semantic line breaks](https://sembr.org/).